src/MsiFileTests/MsiFileTests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Looks like this got missed in the merge- it should be 6.0 now right?

Reply:
Updated to 6.0. Curiously, it insists on using `net6.0-windows`, which wasn't the case for other projects. I'm not 100% why this is needed here but not elsewhere ☹️

Reply:
It's because this project has `UseWPF` set to true, I implemented that feature on my last team 😆